src/modules/Servicehosting/html_admin/mod_servicehosting_index.html.twig

```diff
@@ -220,7 +220,13 @@
                         </div>
                     </div>
                     <div class="mb-3 row">
-                        <label class="form-label col-3 col-form-label">{{ 'Use Secure connection'|trans }}:</label>
+                        <label class="form-label col-3 col-form-label">{{ 'Password length'|trans }}:</label>
+                        <div class="col">
+                            <input class="form-control" type="text" name="passwordlength" value="" placeholder="{{ 'Generated password length. Use blank to use default (10). Used when creating user accounts'|trans }}">
```

Comment:
I'm not a fan of this wording. Perhaps something like this instead:
Length of generated passwords when creating user accounts. Defaults to 10.
Or:
Generated password length. Defaults to 10 when blank. Used when creating user accounts.